### PR TITLE
February UI Maintenace: New Authenticator first-run page, handle "How would you like to sign in" UI

### PR DESCRIPTION
**Pull request by fadidurah**
February UI Maintenace: New Authenticator first-run page, handle "How would you like to sign in" UI

### CODE REVIEW

**Review by melissaahn — APPROVED** [review]

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/microsoftsts/IMicrosoftStsLoginComponentHandler.java [hunk 1]
```diff
@@ -67,4 +67,9 @@ public interface IMicrosoftStsLoginComponentHandler extends IOAuth2LoginComponen
      * interactive request.
      */
     void handleVerifyYourIdentity();
+
+    /**
+     * Handle the How would you like to sign in
+     */
+    void handleHowWouldYouLikeToSignInExpected();
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/microsoftsts/IMicrosoftStsLoginComponentHandler.java]:
```suggestion
    void handleHowWouldYouLikeToSignIn();
```

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java [hunk 2]
```diff
@@ -289,6 +289,23 @@ public static void handleButtonClickSafely(@NonNull final String resourceId) {
         }
     }
 
+    /**
+     * Clicks the button element associated to the supplied resource id with a custom timeout.
+     * Do not throw an exception if the button is not found.
+     *
+     * @param resourceId the resource id of the button to click
+     */
+    public static void handleButtonClickSafely(@NonNull final String resourceId, final long existsTimeout) {
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java:298]: So the tests are not meant to fail with these button clicks not working?

Comment (fadidurah) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java:298]: Yes, we can attempt a button click without failing if we don't find the button.

Comment (iamgusain) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java:298]: nit: we can rename to "handleButtonClickIfExists"

**Review by iamgusain — APPROVED** [review]

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java (hunk 2 above)
Comment (fadidurah) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java:298]: I like this better, but will include it in march maintenance

**Review by shahzaibj — APPROVED** [review]

**Pull request merged**